StepmaniaCore.cmake

```diff
@@ -231,6 +231,9 @@ find_package(Iconv)
 find_package(Threads)
 if (${Threads_FOUND})
   set(HAS_PTHREAD TRUE)
+  list(APPEND CMAKE_REQUIRED_LIBRARIES pthread)
```

Comment:
The only line that bothers me at a glance is this one. Why do we have to manually append pthread here? Won't checking the symbols be enough?